tests/serverpod_test_client/lib/src/protocol/object_field_scopes.dart

```diff
@@ -10,6 +10,7 @@ import 'package:serverpod_client/serverpod_client.dart' as _i1;
 
 class ObjectFieldScopes extends _i1.SerializableEntity {
   ObjectFieldScopes({
+    /// The database ID.
```

Comment:
Removing them is easy, however, I think they are quite helpful when creating a new Object, since vscode shows them when hovering over the named parameter.

I guess it's open for debate :-)

Reply:
Never mind, it seems to be caused by the `this.` syntax.
I'll remove it.

Reply:
removed

Reply:
> I think they are quite helpful when creating a new Object, since vscode shows them when hovering over the named parameter.

This happens when you declare an Object/parameter.